ui/app/routes/vault/cluster/access/method/item/list.js

```diff
@@ -51,11 +51,13 @@ export default Route.extend(ListRoute, {
   },
   setupController(controller) {
     this._super(...arguments);
-    const { apiPath, method, itemType } = this.getMethodAndModelInfo();
+    const { apiPath, method, itemType, methodModel } = this.getMethodAndModelInfo();
     controller.set('itemType', itemType);
     controller.set('method', method);
+    controller.set('methodModel', methodModel);
     this.pathHelp.getPaths(apiPath, method, itemType).then(paths => {
-      controller.set('paths', paths.navPaths.reduce((acc, cur) => acc.concat(cur.path), []));
+      let navigationPaths = paths.paths.filter(path => path.navigation);
+      controller.set('paths', navigationPaths.filter(path => path.itemType.includes(itemType)));
```

Comment:
There's two `filter` calls here - does there need to be?

Reply:
nope, there doesn't. nice catch!